Watchman.Integrations/MongoDB/Session.cs

```diff
@@ -57,6 +57,20 @@ public Session(IMongoDatabase database)
             await this.GetCollection<T>().DeleteOneAsync(x => x.Id == entity.Id);
         }
 
+        public async Task ReplaceByTypeAsync<T>(T item) where T : Entity
```

Comment:
o co tutaj chodzi?

Reply:
to jest dla konfiguracji, ale może się kiedyś przydać również do czegoś innego
chodzi o to, że konfiguracja powinna być tylko jedna zawsze, dlatego w przypadku zmiany ustawień podmieniany jest obiekt na podstawie typu
dałoby się zrobić na zasadzie update czy coś w tym stylu, ale tak wydaje mi się że jest pewniej i łatwiej, bo nie trzeba znać ID poprzedniej konfiguracji w bazie

Reply:
ok, takie coś można zostawić
możliwe że sie przyda, ale musi być odpowiednio zabezpieczone